pkg/infrastructure/validation/validation_test.go

```diff
@@ -607,3 +607,28 @@ func TestCanSkipAlso(t *testing.T) {
 	}
 
 }
+
+func TestFieldNoEmpty(t *testing.T) {
```

Comment:
I think we should build invalid user to check the error message.

Reply:
it is a invalid user already，i check the length and feild of the error.